cmd/cluster-network-check-target/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+func checktargetHandler(w http.ResponseWriter, r *http.Request) {
+	response := os.Getenv("RESPONSE")
+	if len(response) == 0 {
+		response = "Hello OpenShift!"
+	}
+
+	// Echo back the port the request was received on
+	// via a "request-port" header.
+	addr := r.Context().Value(http.LocalAddrContextKey).(net.Addr)
+	if tcpAddr, ok := addr.(*net.TCPAddr); ok {
+		w.Header().Set("x-request-port", strconv.Itoa(tcpAddr.Port))
+	}
```

Comment:
(I presume this was added for network-edge's purposes; we don't need this.)

Reply:
Now that you mention it, you're right: https://github.com/openshift/origin/commit/938a8e99bcbd0d0ca33bf827dfd209859cf525be